Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -198,29 +198,29 @@ public class Session
         //private Session InvalidSession;
 
         //-------------------------------------------------------------------------
-        public bool InitFromInfoOfSessionDetails(SessionDetails session)
+        public bool InitFromInfoOfSessionDetails(SessionDetails Session)
```

Comment:
Parameters should start with lowercase letters.

Reply:
Ahhhh, missed one. I had done a full replace for all "session" to become "Session" before realizing that would affect parameter names. Tried to hit as many of these as I could, will fix!